README.md

```diff
@@ -63,8 +63,6 @@ When connecting a HotRod client to the image, the following SASL properties must
 ```properties
 infinispan.client.hotrod.auth_username=admin
 infinispan.client.hotrod.auth_password=changme
-infinispan.client.hotrod.auth_realm=default
-infinispan.client.hotrod.auth_server_name=infinispan
```

Comment:
Are these configured by default now?

Reply:
The defaults are 

```java
  public static final String DEFAULT_REALM = "default";
   public static final String DEFAULT_SERVER_NAME = "infinispan";
   public static final String DEFAULT_MECHANISM = "SCRAM-SHA-512";
```

the AuthenticationConfigurationBuilder uses those if not provided